src/System.Text.Json/tests/Serialization/Stream.ReadTests.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
nit: Do we need both these using directives? As far as I can tell, we only added `IList<T>`, so the generic one should be enough.

Reply:
I removed `System.Collections` directive and only left there `System.Collections.Generic`, thanks for the suggestion.